pulsar-client-cpp/lib/auth/athenz/ZTSClient.cc

```diff
@@ -143,7 +143,7 @@ namespace pulsar {
         size_t length = strlen(input);
         char *result = (char*)malloc(length);
 
-        bio = BIO_new_mem_buf(input, -1);
+        bio = BIO_new_mem_buf((void *)input, -1);
```

Comment:
Shouldn't this be `const void *` ?

Reply:
https://gist.github.com/superwills/5415344#file-openssl-rsa-encryption-sample-L28
https://linux.die.net/man/3/bio_new_mem_buf